internal/gen/connect/ping/v1/ping.pb.go

```diff
@@ -12,12 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// The canonical location for this file is
+// https://github.com/bufbuild/connect-go/blob/main/internal/proto/connect/ping/v1/ping.proto.
+
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
 // 	protoc        (unknown)
 // source: connect/ping/v1/ping.proto
 
+// The connect.ping.v1 package contains an echo service designed to test the
```

Comment:
I'm guessing nothing has changed to the generated code between v1.28.1 and v1.30.0 (besides the version number), but it might be good to confirm the comment handling hasn't changed.

Reply:
Good point, I'll update.

Reply:
No changes in 1.30. I'm going to hold off on updating our `go.mod` - I'll open an issue discussing whether we ought to be aggressively bumping library versions here, since that effectively _forces_ users to update.